scanner/templates/go/Dockerfile

```diff
@@ -1,14 +1,14 @@
 ARG GO_VERSION=1
-FROM golang:${GO_VERSION}-alpine as builder
+FROM golang:${GO_VERSION}-bookworm as builder
 
 WORKDIR /usr/src/app
 COPY go.mod go.sum ./
 RUN go mod download && go mod verify
 COPY . .
-RUN go build -v -o /run-app .
+RUN CGO_ENABLED=1 go build -v -o /run-app .
```

Comment:
CGO_ENABLED=1 is the default for non-alpine images

Reply:
oh, ironically I put that there just so I could copy/paste it for somewhere else! bah - I'll keep it off.